packages/grafana-e2e/src/flows/addDataSource.ts

```diff
@@ -32,7 +32,14 @@ export const addDataSource = (config?: Partial<AddDataSourceConfig>): string =>
   form();
   e2e.pages.DataSource.saveAndTest().click();
   e2e.pages.DataSource.alert().should('exist');
-  e2e.pages.DataSource.alertMessage().should('contain.text', expectedAlertMessage);
```

Comment:
@stevenvachon did this not work https://github.com/cypress-io/cypress/issues/3747#issuecomment-605457187

Reply:
It does, actually. I think that it's not obvious that it's an assertion, however.

Reply:
Went with it, and added a comment.